External/FEXCore/Source/Interface/Core/JIT/Arm64/JIT.cpp

```diff
@@ -1359,14 +1351,18 @@ void *JITCore::CompileCode([[maybe_unused]] FEXCore::IR::IRListView<true> const
         auto Desired = GetSrc<RA_64>(Op->Header.Args[1].ID());
         auto MemSrc = GetSrc<RA_64>(Op->Header.Args[2].ID());
 
-        add(TMP1, MEM_BASE, MemSrc);
+        if (!CTX->Config.UnifiedMemory) {
```

Comment:
Do we support non-unified memory?

Design wise, wouldn't it be simpler to have membase as IR, and not in the backend?

Reply:
Currently non-unified is required for the unit tests since they use a bunch of absolute addressing that causes issues with unified. Otherwise non-unified is effectively not implemented (Especially after this PR which strips it out of syscalls entirely)